pra_request_tracker/record_requests/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from pra_request_tracker.users.models import User
+
+
+class RecordRequest(models.Model):
+    requester = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    description = models.TextField()
+    agency = models.CharField(max_length=100)
```

Comment:
Probably 256 for this IMO

And also, maybe this should be another model?? 😮

Reply:
Definitely should be another model